docs/DeveloperGuide.md

```diff
@@ -238,6 +238,38 @@ _{more aspects and alternatives to be added}_
 
 _{Explain here how the data archiving feature will be implemented}_
 
+### Mark/Unmark Feature
+This feature allows users to mark the orders as complete/incomplete.
+#### Implementation
+The mark feature consists of three commands, `MarkCommand` and `UnmarkCommand`.
```

Comment:
Late to the party, but this doesn't look right. Do you mean comprises of two commands?